pages/team.js

```diff
@@ -175,14 +175,14 @@ When not seen busy juggling between different tasks he takes up, he can be seen
                 img="https://github.com/bellesea.png"
                 pronouns="she/her"
               />
-              {/*<Bio
+              <Bio
                 name="Kunal Botla"
                 teamRole="Bank Operations"
-                text={`yeah`}
-                img="/stickers/logo.png"
+                text={`Kunal love to make to make. Starting Project Boom to help provide computers, helping build and run Hack Club Bank, and organizing MAHacks for a post-pandemic world. He takes photos to tell stories of an everchanging world.`}
+                img="https://pbs.twimg.com/profile_images/1479510388790206478/Yvf50MUV.jpg"
```

Comment:
One other thing: I don't have too much trust in twitter's image urls, could we use smth like cdn?

Reply:
You could also use `/hackers/kunal.jpg` here instead of the Twitter CDN.

Reply:
good point; i'll make that edit too

Reply:
I changed it to https://kunalbotla.com/images/kunal.jpeg so I can update a single place over time